packages/gafl-webapp-service/package.json

```diff
@@ -36,7 +36,7 @@
     "prepare": "gulp --gulpfile build/gulpfile.cjs"
   },
   "dependencies": {
-    "@defra-fish/business-rules-lib": "^0.0.1-beta.28",
+    "@defra-fish/business-rules-lib": "0.0.1-beta.28",
```

Comment:
Shouldn't modify these as they are managed by lerna and modified each time a change is pushed to master.